Enable GuardDuty and SecurityHub (#4174) (#4629)
DataBiosphere/azul: UPGRADING.rst

have too many entries in this file.

#4174 Enable GuardDuty and SecurityHub
======================================

This change enables AWS Config, GuardDuty, and SecurityHub, deployed as part of

> the ``shared`` terraform component. Prior to deploy the operator must ensure

Reviewer: ```suggestion
the ``shared`` Terraform component. Prior to deploy the operator must ensure
```

Reviewer: grammar

these services are not currently active and disable/remove any that are. Use the

> ``aws`` "describe" and "list" commands to obtain the status of each service, and
> the "delete" and "disable" commands to remove the ones that are active ::

Reviewer: ```suggestion
AWS CLI's _list_ and _describe_ functionality to obtain the status of each service, and
```

And in kind for the next line.
